dashboards/grafana-dashboard-insights-yuptoo-general.configmap.yaml

```diff
@@ -0,0 +1,661 @@
+apiVersion: v1
+data:
+  dashboard.json: |-
+    {
+      "__inputs": [
+        {
+          "name": "DS_CRCS02UE1-PROMETHEUS",
```

Comment:
Would this be configurable through grafana UI to switch from stage to prod ?

Reply:
Yes